packages/mdc-toolbar/component.ts

```
@@ -65,8 +56,25 @@ export class MDCToolbar extends MDCComponent {
     super.destroy();
   }
 
+  set fixedAdjustElement(element: HTMLElement | null) {
+    this.fixedAdjustElement_ = element;
+    this.foundation_.updateAdjustElementStyles();
+  }
+
+  get fixedAdjustElement(): HTMLElement | null {
+    return this.fixedAdjustElement_;
+  }
+
+  private get firstRowElement_(): HTMLElement {
```

isn't it also accurate that this could return both `HTMLElement | null`?

---

From the README, it looks like a row element is always present. Is that not the case?

---

It should be...I was thinking we should throw another error if it doesn't exist, but is that the only required element in this component?

---

Yeah it looks like this is the only required subelement

---

let's throw an error then if the firstRowElement is not supplied. Then we can definitely say that this getter will always return an HTMLElement.

---

Good call. Done. I also made this a normal property rather than a getter for consistency with other packages.